src/ISA/ISA.Spreadsheet/ArcTable.fs

```diff
@@ -0,0 +1,53 @@
+﻿module ISA.Spreadsheet.ArcTable
+
+open ISA
+open FSharpAux
+open FsSpreadsheet
+
+let annotationTablePrefix = "annotationTable"
```

Comment:
This should be unified between Swate, Swate-standalone and ISA.Spreadsheet. 

In addition remember to make all strings like this to literals

Reply:
Any idea about where to put this? I could put this together with the `tryFromFsWorksheet: FsWorksheet -> FsTable option` function you mentioned below into a `AnnotationTable.fs` and module.

Reply:
Maybe put it on some member of `ArcTable` ? `ArcTable.toAnnotationTableName()` ?